pubspec.yaml

```diff
@@ -16,7 +16,7 @@ dependencies:
   decimal: ^2.1.0
   dio: ^4.0.0
   faker: ^2.0.0
-  firebase_core: ^1.13.1
+  firebase_core: ^1.16.0
```

Comment:
Interesting. I had to fix this yesterday and I definitely didn't upgrade to Flutter v3.0.0.